src/wkt_macro.rs

```diff
@@ -721,64 +296,44 @@ macro_rules! multi_polygon_zm {
 #[doc(hidden)]
 macro_rules! geometry_collection {
     (EMPTY) => {
-        GeometryCollection::empty(Dimension::XY)
+        $crate::types::GeometryCollection::empty(Dimension::XY)
```

Comment:
Does this need to be 
```suggestion
        $crate::types::GeometryCollection::empty($crate::types::Dimension::XY)
```